### PR TITLE
RTC-10351 - Adding optional className to TextEllipsis

### PR DESCRIPTION
**Pull request by axeleriksson147**
- Added support for className in TextEllipsis
- Added initial support for data-testid to Dropdown component. This needs to be worked on more so all props can be passed correctly but it was a quite tricky component to navigate so I think this is an okay start.

### CODE REVIEW

File: src/components/text-ellipsis/TextEllipsis.tsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ interface TextEllipsisProps extends Omit<React.HTMLProps<HTMLDivElement>, 'type'
     /** Text that should be ellipsed */
     children?: React.ReactNode;
 
+    /** Optional CSS class name */
+    className?: string;
```
Comment (symphony-adnane) [inline src/components/text-ellipsis/TextEllipsis.tsx:10]: thought there would come from the extended props no ?

File: stories/Dropdown.stories.tsx [hunk 2]
```diff
@@ -239,6 +239,7 @@ export const Select: React.FC = () => (
     <h2 className="tk-mt-5h">Scroll into view - disabled</h2>
     <p>{"When pressing the dropdown while it's not fully in view, it will not scroll into view"}</p>
     <Dropdown
+      data-testid="RTC_SETTINGS_DROPDOWN__AUDIO"
```
Comment (symphony-adnane) [inline stories/Dropdown.stories.tsx:242]: could we remove this :)

Comment (axeleriksson147) [reply stories/Dropdown.stories.tsx]: Yes, forgot while testing. Fixed!

File: src/components/text-ellipsis/TextEllipsis.tsx (hunk 1 above)
Comment (axeleriksson147) [reply src/components/text-ellipsis/TextEllipsis.tsx]: Hm, yes, I've made a mistake.

Removed the prop but kept the part where I put it inside the className={ classnames(...) } property.

**Review by symphony-adnane — APPROVED** [review]

**Review by AliciaSymphony — APPROVED** [review]

**Pull request merged**